portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -31,32 +33,45 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  //initialize the number of comments here so that the system always remembers even after refreshing with default 5
```

Comment:
Nit: sentence case for comments (meaning capitalize the first word) + space between // and first letter